bot/cogs/coding/coding.py

```diff
@@ -129,7 +135,9 @@ async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[N
                 compilerFlags.extend(line[15:].strip("`").split(" "))
 
             elif line.startswith("command-line-options "):
-                commandLineOptions.extend(line[21:].strip("`").split(" "))  # cli options
+                commandLineOptions.extend(
+                    line[21:].strip("`").split(" ")
+                )  # cli options
```

Comment:
```suggestion
                commandLineOptions.extend(line[21:].strip("`").split(" ")) 
```
This should also not be split, and the comment can be removed, cli options doesn't say much and we can already see that from the function name